new-dti-website/components/navbar.tsx

```diff
@@ -77,13 +77,15 @@ const Navbar: React.FC = () => {
               alt="Hamburger Menu Icon"
               onClick={(e) => handleMenuClick()}
             />
+          ) : (
+            <div className="h-12 w-auto md:h-14" />
           )}
         </div>
       </div>
       {isMenuOpen && (
-        <div className="fixed inset-0 bg-black bg-opacity-25 backdrop-blur-sm z-40 lg:hidden">
-          <div className="w-full h-full flex flex-col items-end">
-            <div className="p-4">
+        <div className="fixed inset-0 bg-black bg-opacity-25 lg:hidden">
```

Comment:
Thoughts on keeping the backdrop blur? There's now a harsh, but less noticeable, transition in the navbar.
![image](https://github.com/user-attachments/assets/470bdc63-9a43-4de8-9dcc-7ed113f3cfc8)

Reply:
You mean remove the black bar entirely and just keep the original background?

Reply:
Hmm.. I meant that the background behind the cornell dti logo on top should also be blurred, so we don't get the same harsh transition in this screen.

Reply:
Oh hmm, I was following the design which deliberately did not blur the logo, I can see how it would make sense to get rid of the black bar there. I'll double check with Vannessa.